index.js

```diff
@@ -61,6 +148,28 @@
  *
  */
 
+var homework = {
+	q0: countdown(),
+	q1: max(),
+	q2: longer(),
+	q3: isEven(),
+	q4: getAreaOfCircle(),
+	q5: getFullName(),
+	q6: reverseString(),
+	q7: maxArray()
+}
+
+
  /**
   * Q9: Use the Object from Q8 to call all of the functions you've created.
   */
+
+homework.countdown();
```

Comment:
The reason that this didn't work is because we already called the function above. Unless those functions return another function (none of these do), then this will throw an error.